### PR TITLE
Use new version of elasticsearch library

### PR DESCRIPTION
**Pull request by rahulPalliyalil**
### What

This pr attempts to update the dp-elasticsearch library to the latest version and also point the dp search data importer to new version of elasticsearch 7.10

### How to review

Check if the changes make sense. 

### Who can review

Anyone except me

### CODE REVIEW

File: service/service_test.go [hunk 1]
```diff
@@ -242,8 +219,15 @@ func TestClose(t *testing.T) {
 
 		hcStopped := false
 
-		funcElasticSearchMockOk := func(ctx context.Context, cfg *config.Config) (*dpElasticSearch.Client, error) {
-			return nil, nil
+		esDestURL := "http://locahost:9999"
+
+		elasticSearchMock, _ := dpElastic.NewClient(dpESClient.Config{
```
Comment (nshumoogum) [inline service/service_test.go:224]: Same as above

**Review by nshumoogum — APPROVED** [review]

**Review by nshumoogum — CHANGES_REQUESTED** [review]
2 comments to be looked at before approval

File: service/service_test.go [hunk 2]
```diff
@@ -45,44 +47,19 @@ var (
...
-}
-
 func TestRun(t *testing.T) {
 
 	Convey("Given a set of mocked dependencies", t, func() {
 
-		esDestURL := "http://locahost:9999"
-		httpCli := clientMock(doFuncWithValidResponse)
-		elasticSearchMock := dpElasticSearch.NewClientWithHTTPClient(esDestURL, false, httpCli)
+		elasticSearchMock, _ := dpElastic.NewClient(dpESClient.Config{
```
[the remark this answers is not in the document]
Comment (rahulPalliyalil) [reply service/service_test.go]: Done.

File: service/service_test.go (hunk 1 above)
Comment (rahulPalliyalil) [reply service/service_test.go]: Done.

**Pull request merged**